### PR TITLE
WFLY-14480 Fix duplicate declaration of version of io.smallrye.reacti…

### PR DESCRIPTION
**Pull request by rhusar**
…ve:smallrye-reactive-converter-api:jar + wrong version + duplicate jboss module

Resolves
https://issues.redhat.com/browse/WFLY-14480
https://issues.redhat.com/browse/WFLY-14483

### CODE REVIEW

**Comment by rhusar** [comment]
@kabir Please review.

**Comment by kabir** [comment]
@rhusar I am getting some failures locally when trying this. I'll have a play and see if I can fix it

**Comment by rhusar** [comment]
@kabir I see that too - the only change that should be affecting these is version of `smallrye-reactive-converter-api` being bumped from 1.2 -> 2.1. If this is a real problem, I think I can have MP FT use the old version.

File: pom.xml [hunk 1]
```diff
@@ -314,17 +314,15 @@
...
         <version.io.smallrye.opentracing>2.0.0-RC1</version.io.smallrye.opentracing>
-        <version.io.smallrye.reactive-utils>2.0.0</version.io.smallrye.reactive-utils>
+        <version.io.smallrye.reactive-utils>2.1.1</version.io.smallrye.reactive-utils>
         <version.io.smallrye.smallrye-common>1.5.0</version.io.smallrye.smallrye-common>
         <version.io.smallrye.smallrye-config>2.0.2</version.io.smallrye.smallrye-config>
         <version.io.smallrye.smallrye-fault-tolerance>5.0.0</version.io.smallrye.smallrye-fault-tolerance>
         <version.io.smallrye.smallrye-health>3.0.0</version.io.smallrye.smallrye-health>
         <version.io.smallrye.smallrye-jwt>3.0.0</version.io.smallrye.smallrye-jwt>
         <version.io.smallrye.smallrye-metrics>3.0.1</version.io.smallrye.smallrye-metrics>
         <version.io.smallrye.smallrye-mutiny>0.13.0</version.io.smallrye.smallrye-mutiny>
```
Comment (kabir) [inline pom.xml:324]: @rhusar Upgrade mutiny to 0.14.0 from 0.13.0 and it should be fine. It seems to be the version expected by https://github.com/smallrye/smallrye-reactive-utils/blob/2.1.1/pom.xml#L47

Comment (rhusar) [reply pom.xml]: Done.

**Comment by rhusar** [comment]
retest this please

**Pull request merged**